tests/steps/test-report.sh

```diff
@@ -58,6 +58,38 @@ echo "👍🏻 A PDF report generated correctly"
     while IFS= read -r t; do
         metric=$(echo "${t}" | cut -f1 -d' ')
         echo "${metric}" | grep '^\\item\\ff{[a-zA-Z0-9-]\+}:$' > /dev/null
-    done < "${TARGET}/temp/list-of-metrics.tex"
+    done < "${TARGET}/temp/list-of-metrics.tex.unstructured"
 } > "${stdout}" 2>&1
 echo "👍🏻 A list of metrics is properly formatted"
+
+{
+    test_metric_sh="#!/bin/bash\n\n"
+    test_metric_sh+="output=\$(realpath \"\$2\")\n"
+    test_metric_sh+="for idx in {2..5}; do\n"
+    test_metric_sh+="    echo \"Test-\${idx} 0 [Test group \$((idx % 2))] Test metrics\" >> \"\${output}\"\n"
+    test_metric_sh+="done\n"
+    printf "%b" "$test_metric_sh" > "${LOCAL}/metrics/group_test.sh"
```

Comment:
@Raleksan this is pretty dangerous. The file may easily be committed to the repository, since you create in inside the "live" part of the code base (which is a [bad practice](https://stackoverflow.com/questions/28253556/what-is-a-right-way-to-deal-with-tests-temporary-files-and-version-control)).

Reply:
@yegor256 

I can implement several solutions, could you please tell me which one is better:
- Delete `group_test.sh` after running `steps/report.sh` for this test
- Add `group_test.sh` to `.gitignore`.
- Add the test metrics to the directory with the metrics already generated while `steps/report.sh` is running. This method will require tools like [inotify](https://linux.die.net/man/1/inotifywait) to control the deletion and creation of the `temp` directory.

Reply:
@Raleksan I would do it differently. In the `report.sh` I would say something like this (at the beginning):

```
if [ -z "LOCAL_METRICS" ]; then
  LOCAL_METRICS=${LOCAL}/metrics
fi
```

Then, in the test, I would call it like this:

```
LOCAL_METRIC=your-test-directory "${LOCAL}/steps/report.sh"
```

Thus, you will fool the script, but won't hurt it.

Reply:
@yegor256 

There was no such feature, but I added it and used it for testing purposes.